requirements.txt

```
@@ -1,3 +1,5 @@
 requests==2.31.0
 datasets==2.14.5
 Pillow==10.0.1
+huggingface_hub
```

```suggestion
huggingface_hub>=0.18
```

You need to pin to at least `0.18.0` to get `preupload_lfs_files`